numpy/core/fromnumeric.py

```diff
@@ -890,6 +890,13 @@ def argsort(a, axis=-1, kind='quicksort', order=None):
     array([[0, 1],
            [0, 1]])
 
```

Comment:
Can you add a short description here to explain the example? That would also be appreciated in the other examples.